tools/project_manager/projectmgr.js

```diff
@@ -1,12 +1,18 @@
-/*global moment serverTimezone*/
+/*global serverTimezone*/
 
 window.addEventListener("DOMContentLoaded", () => {
     "use strict";
 
-    var timeSpan = document.getElementById("server-time");
+    const timeSpan = document.getElementById("server-time");
+    const dateTimeFormat = new Intl.DateTimeFormat('en-US', {
```

Comment:
This needs to match the user's locale so French users get this in French. You probably want `get_user_language()` but that uses an underscore instead of a dash (eg: `en_US` vs `en-US`).